Commands/Public/contacts.js

```diff
@@ -227,7 +234,11 @@ module.exports = async(client, msg, suffix) => {
 			msg.author.busy = false;
 			if (/^9$/.test(collected.first().content)) return contactList();
 			if (!collected.first() || /^0$/.test(collected.first().content)) return;
-
+			if (collected.first.length() < 0 || collected.first.length() > 100) {
+				let isItMax = collected.first.length() > 100;
+				msg.author.busy = false;
+				return msg.channel.send({ embed: { title: `Description ${isItMax ? "too long" : "too short"}`, description: `Please ${isItMax ? "lengthen" : "shorten"} the description.` } });
```

Comment:
The same thing, turn it into a feedback loop.

Reply:
that is planned

Reply:
I just noticed that this is checking if the message is too long... and then tells the user to lengthen it 🤦‍♂️